.github/workflows/update-dependencies.yml

```
@@ -11,14 +11,10 @@ on:
 jobs:
   update-dependencies:
     name: Update dependencies
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

```suggestion
    runs-on: ubuntu-latest
```

it was pinned to 20.04 for python 3.6 and can be moved to latest now.